api/controllers/Depmu_EntryController.js

```diff
@@ -39,7 +39,7 @@ module.exports = {
         _.merge(prebooking, result)),
 
   filterContingencyWithoutCid: prebooking =>
-  !prebooking.contingency || (prebooking.contingency && _.isNull(prebooking.cid_id)),
+  !prebooking.contingency || prebooking.contingency && _.isNull(prebooking.cid_id),
 
```

Comment:
you should find that is the boolean logic i'm expressing, the added brackets weren't needed which is why it doesn't fail any unit tests